mandatory/1-syntax-errors.js

```diff
@@ -1,16 +1,23 @@
 // There are syntax errors in this code - can you fix it to pass the tests?
 
 function addNumbers(a b c) {
-  return a + b + c;
+  return a + b + c;}
+  function addNumbers(a, b, c) {
+    return  a  +  b  +  c ;
 }
 
 function introduceMe(name, age)
   return "Hello, my name is " + name "and I am " age + "years old";
+  function introduceMe(name, age) {
+    return "Hello, my name is " + name + " and I am " + age + " years old";
+  }
 
 function getTotal(a, b) {
   total = a ++ b;
+  total = a + b;
 
   return "The total is total";
+  return "The total is " + total;
```

Comment:
This line of code will never be ran. If you look at it in VsCode, it's slightly lighter than the code around it so you can easily tell that it's redundant